src/components/raycaster.js

```diff
@@ -216,11 +218,13 @@ module.exports.Component = registerComponent('raycaster', {
       }
 
       // Emit intersection cleared on both entities per formerly intersected entity.
+      clearedIntersectedEls.length = 0;
       for (i = 0; i < prevIntersectedEls.length; i++) {
         if (intersectedEls.indexOf(prevIntersectedEls[i]) !== -1) { return; }
-        el.emit('raycaster-intersection-cleared', {el: prevIntersectedEls[i]});
```

Comment:
This will fail only for GearVR and Daydream, because other controllers don't use raycaster, but sphere collider.
I guess in super-hands, we need to get the raycaster component with
`var raycaster = this.el.querySelector('[raycaster]')` and iterate over raycaster.clearedIntersectedEls. But I'm not sure what do to with your if condition @wmurphyrd  to detect if it's a raycaster event or an sphere-collider event.

Reply:
Let's continue the discussion there https://github.com/wmurphyrd/aframe-super-hands-component/issues/79

Reply:
Yeah, you'd iterate over them and clear them all at once versus one at a time.